e2e/rbd.go

```diff
@@ -1028,9 +1032,10 @@ var _ = Describe("RBD", func() {
 						defaultSCName,
 						nil,
 						map[string]string{
-							"imageFeatures": "layering,journaling,exclusive-lock",
-							"mounter":       "rbd-nbd",
-							"mapOptions":    nbdMapOptions,
+							"imageFeatures":   "layering,journaling,exclusive-lock",
+							"mounter":         "rbd-nbd",
+							"mapOptions":      nbdMapOptions,
+							"cephLogStrategy": "preserve",
```

Comment:
define a const for preserve and reuse it.

Reply:
done